docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,107 @@
+# Coverage Info
+
+The {ref}`--coverage_info` flag enables automatic computation of coverage
+information for a verification run. In particular, using this flag can help you
+answer questions such as:
+
+* *Are all solidity functions from the input involved in proving my rules?*
+* *Are all solidity commands from the input involved in proving my rules?*
+* *Supposing an `assert` in my rule is not reachable, what is the reason for the
+  unreachability?*
+* *Do I really need all hooks that are defined in my `.spec` file(s)?*
+* *Do I really need all `require` statements in my rule?*
+* *Do I really need to initialize a CVL variable in my rule?*
+* *Do I really need all of the preserved blocks in my CVL `invariant`?*
+
+To answer the above questions, the Certora Prover identifies a minimal subset of
+the commands in the input `.sol` and `.spec` files that are relevant for proving
+the CVL properties. If some of the input `.sol` commands are not relevant for
+deriving the proof, it might indicate that the specification does not cover all
+behavior implemented in the smart contract. If some of the input `.spec`
```

Comment:
I'd generally refer to `.sol` files or Solidity commands but not `.sol` commands.  Similarly "`.spec` commands" should be "CVL commands"